### PR TITLE
Dark Mode: class and mixin documentation - RSC-1577

### PR DESCRIPTION
**Pull request by rpokorny**
https://issues.sonatype.org/browse/RSC-1577

### CODE REVIEW

File: gallery/src/styles/DarkMode/DarkModeDarkOverrideExample.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE HTML>
+<html class="nx-html nx-html--page-scrolling nx-html--enable-color-schemes nx-html--dark-mode">
+  <head>
+    <title>Dark Mode Example: No Opt-In</title>
```
Comment (EmilyBermudez88) [inline gallery/src/styles/DarkMode/DarkModeDarkOverrideExample.html:4]: This example does opt-in to dark mode

File: gallery/src/styles/DarkMode/DarkModeClassPage.tsx [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+          </NxTable.Head>
+          <NxTable.Body>
+            <NxTable.Row>
+              <NxTable.Cell><NxCode>nx-html--enable-color-schemes</NxCode></NxTable.Cell>
+              <NxTable.Cell>
+                This class must be set on the <NxCode>&lt;html&gt;</NxCode> element in order to opt in to
+                any non-default theming. If this class is not set, the light theme will always be used.
+                If this class is set, the user's theme preference determined by
+                the <NxCode>prefers-color-scheme</NxCode> media query will be used in conjunction with
+                the <NxCode>nx-html--dark-mode</NxCode> and <NxCode>nx-html--dark-mode</NxCode> classes to determine
```
Comment (EmilyBermudez88) [inline gallery/src/styles/DarkMode/DarkModeClassPage.tsx:46]:
```suggestion
                the <NxCode>nx-html--dark-mode</NxCode> and <NxCode>nx-html--light-mode</NxCode> classes to determine
```

File: gallery/visualtests/darkModeSettings.js
[the remark this answers is not in the document]
Comment (EmilyBermudez88) [reply gallery/visualtests/darkModeSettings.js:1]: > As I implemented the other visual tests I realized that they probably made these tests redundant; they cover the same functionality in a simpler and more thorough way.

I agree, the new tests are incredibly thorough in comparison to what was there before!

**Review by EmilyBermudez88 — APPROVED** [review]
Minus the few minor documentation changes, it looks great

File: gallery/src/styles/DarkMode/DarkModeDarkOverrideExample.html (hunk 1 above)
Comment (rpokorny) [reply gallery/src/styles/DarkMode/DarkModeDarkOverrideExample.html]: > This example does opt-in to dark mode (same for DarkModeLightOverrideExample and DarkModeOptInExample)

Fixed all of the copy/pasted example titles in 843efbafa

**Pull request merged**